NLogPerformance - Added Logger-Name as command line argument

snakefoot opened the pull request:
Allow more tests with the same nlog.config

snakefoot committed:
NLogPerformance - Added Logger-Name as command line argument. Allow more tests with the same nlog.config

304NotModified commented on NLogPerformance/Program.cs:
```diff
@@ -51,13 +66,13 @@ static void Main(string[] args)
-            Console.WriteLine("| {0,10} | {1,4} | {2,7} | {3,7} |", _messageCount, _messageSize, _threadCount, _loggerCount);
+            Console.WriteLine("| Logger Name      | Messages   | Size | Threads | Loggers |");
+            Console.WriteLine("|------------------|------------|------|---------|---------|");
+            Console.WriteLine("| {0,-16} | {1,10} | {2,4} | {3,7} | {4,7} |", _loggerName, _messageCount, _messageSize, _threadCount, _loggerCount);
```
please use ` {1,10:N0} `

I don't like reading 1000000 while it could be 1.000.000.

I added them in https://github.com/NLog/NLog-Performance-tests/pull/10/files, but those are lost :(

304NotModified replied:
Also G3 gives +e03 etc. It should be N3 IMO

snakefoot commented:
I have fixed the formatting of the message-count. The other formatting options are unchanged (and not lost). I have just added the LoggerName to the parameters table.

304NotModified commented:
oops forgot about this one!
